CHANGELOG.md

```diff
@@ -7,6 +7,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 [Unreleased]
 
+## [2.1.1] - 2024-06-18
+
+### Fixed
+- `@rarimo/rarime-connector` - request params
```

Comment:
Add a proper description of what was fixed, including the MM version that introduced the change